src/app/views/teams/team-create/CreateTeam.jsx

```diff
@@ -0,0 +1,175 @@
+        } else {
+            // Can leave
+            return true;
+        }
+    };
+
+    const handleTeamNameChange = event => {
+        setTeamName(event.target.value);
+    };
+
+    const requestCreateTeam = () => {
+        if (newTeam.unsavedChanges && teamName !== "") {
+            // All user created teams will have an equal precedence of 10. 0-9 are for 'roles' 11-99 are unused.
+            const body = {
+                name: teamName,
+                precedence: 10,
+            };
+            dispatch(createTeam(body));
+        } else {
+            const notification = {
+                type: "warning",
+                message: "Unable to save team, you need to at least give the team a name",
+                isDismissable: true,
+            };
+            notifications.add(notification);
+        }
+    };
+    let contentActionBarProps = {
+        buttons: [
+            {
+                id: "create-team-btn",
+                text: "Create Team",
+                interactionCallback: requestCreateTeam,
+                style: "positive",
+                disabled: false,
+            },
+        ],
+        cancelCallback: () => {
+            const previousUrl = url.resolve("../", true);
+            dispatch(push(previousUrl));
+        },
+        stickToBottom: true,
+        unsavedChanges: newTeam.unsavedChanges,
+    };
+    let teamsMemberChips = (
+        <div className="chip__container chip__container--gap-10">
+            {newTeam.usersInTeam?.map((teamMember, i) => {
+                return (
+                    <Chip
+                        key={`teamMember-${i}`}
+                        icon="person"
+                        style="standard"
+                        text={`${teamMember.forename} ${teamMember.lastname}`}
+                        removeFunc={() => {
+                            let userToRemove = newTeam.allUsers?.find(viewer => viewer.email === teamMember.email);
+                            dispatch(removeUserFromNewTeam(userToRemove));
+                        }}
+                    />
+                );
+            })}
+        </div>
+    );
+    const noTeamMembers = <p>This team has no members</p>;
+    const teamNameInputArea = <Input id="team-name-id" label="Name" type="text" onChange={handleTeamNameChange} />;
+
+    return (
+        <div className="grid grid--justify-space-around">
+            <div className="grid__col-4 margin-top--1 margin-bottom--1">
+                <span>
+                    &#9664; <Link to={url.resolve("../")}>Back</Link>
+                </span>
+                <h1 className="margin-top--1 margin-bottom--1">Create a preview team</h1>
+                {teamNameInputArea}
+                <span>
+                    <strong>Members</strong>
+                </span>
+                {newTeam.usersInTeam?.length > 0 ? teamsMemberChips : noTeamMembers}
+            </div>
+            <UsersNotInTeam loading={true} />
```

Comment:
whys this is loading true always, if its not managed by parent certainly do not need to be passed ?